misk/src/main/kotlin/misk/web/MiskWebModule.kt

```diff
@@ -119,6 +123,12 @@ class MiskWebModule @JvmOverloads constructor(
       ServiceModule<JettyConnectionMetricsCollector>()
         .enhancedBy<ReadyService>()
     )
+    install(
+      ServiceModule<ReadinessCheckService>()
+        .enhancedBy<ReadyService>()
```

Comment:
Was going to say we should probably use `dependsOn` and shut this down as soon as we get sigterm but [apparently](https://github.com/kubernetes/kubernetes/issues/107473) k8s probes after shutdown are a little flaky. Maybe add that issue in a comment?

Reply:
This `enhancedBy` is a commonality for all services in this module. Are you worried about its usage here specifically, or just the general lack of dependency?